scalafix-rules/src/main/scala/scalafix/internal/rule/RemoveUnused.scala

```diff
@@ -27,15 +27,17 @@ class RemoveUnused(config: RemoveUnusedConfig)
   override def isRewrite: Boolean = true
 
   private def warnUnusedPrefix = List("-Wunused", "-Ywarn-unused")
+  private def warnUnusedString = List("-Xlint", "-Xlint:unused")
   override def withConfiguration(config: Configuration): Configured[Rule] = {
     val hasWarnUnused = config.scalacOptions.exists(option =>
-      warnUnusedPrefix.exists(prefix => option.startsWith(prefix))
+      warnUnusedPrefix.exists(prefix => option.startsWith(prefix)) ||
+        warnUnusedString.contains(option)
     )
     if (!hasWarnUnused) {
       Configured.error(
         s"""|The Scala compiler option "-Ywarn-unused" is required to use RemoveUnused.
             |To fix this problem, update your build to use at least one Scala compiler
-            |option that starts with -Ywarn-unused or -Wunused (2.13 only)""".stripMargin
+            |option like -Ywarn-unused, -Xlint:unused (2.12.2 or above), or -Wunused (2.13 only)""".stripMargin
```

Comment:
Unused warnings by `-Xlint` or `-Xlint:unused` are effective since Scala 2.12.2.

https://github.com/scala/bug/issues/10270
https://github.com/scala/scala/pull/5402

So suggesting `-Xlint:unused` is not relevant in Scala 2.12.1 or lower while Scalafix still supports 2.11 . To be safer, I would clarify which versions are supported.